Seminar/Homework/1-4.py

```diff
@@ -0,0 +1,4 @@
+#Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов, а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+s= int(input('Введите общее количество журавликов: '))
```

Comment:
Неплохо бы проверять входные данные на кратность 6 потому что только в этом случае мы можем получить адекватный результат.
```suggestion
s = int(input('Введите общее количество журавликов: '))
```